axlearn/common/flash_attention/gpu_attention_test.py

```diff
@@ -152,6 +153,8 @@ def test_decode_against_ref(
         kv_head_factor: int,
         window_len: int,
     ):
+        if jax.default_backend() != "gpu" and seq_len > 1024:
```

Comment:
Nit: can we check it against "cpu" directly instead of `!= "gpu"`?

Reply:
Yes, done.